packages/core/src/package.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
Something wrong happened here. This is the `package.json` that should be in the root folder

Reply:
👍 removed in cf7f466